homeassistant/components/tile/device_tracker.py

```diff
@@ -59,16 +57,6 @@ def __init__(self, entry: ConfigEntry, coordinator: TileCoordinator) -> None:
         self._attr_unique_id = f"{entry.data[CONF_USERNAME]}_{self._tile.uuid}"
         self._entry = entry
 
-    @property
-    def available(self) -> bool:
-        """Return if entity is available."""
-        return super().available and not self._tile.dead
-
-    @property
-    def device_info(self) -> DeviceInfo:
-        """Return device info."""
-        return DeviceInfo(identifiers={(DOMAIN, self._tile.uuid)}, name=self._tile.name)
-
     @callback
```

Comment:
I think move `_handle_coordinator_update()` also to the base entity and an abstract `_update_from_latest_data()`

Reply:
I want to implement other platforms like binary, but they will not use the `_handle_coordinator_update()`

Reply:
I would assume they would also use the coordinator but if that's not the case let's leave it then 👍

Reply:
Oh they will use the coordinator, but they won't use a separate function to set the state